mvc/views/tag-helpers/intro.rst

```diff
@@ -1,8 +1,281 @@
+
+Using ``@tagHelperPrefix`` to make Tag Helper usage explicit
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``@tagHelperPrefix`` directive allows you to specify a tag prefix string to enable Tag Helper support and to make Tag Helper usage explicit. In the code image below, the Tag Helper prefix is set to "th:", so only those elements using the prefix "th:" support Tag Helpers (Tag Helper-enabled elements have a distinctive font). The ``<label>`` and ``<span>`` elements have the Tag Helper prefix and are Tag Helper-enabled, while the ``<input>`` element does not.
+
+.. image:: intro/_static/thp.png 
+
+The same hierarchy rules that apply to ``@addTagHelper`` also apply to ``@tagHelperPrefix``.
+
+IntelliSense support for Tag Helpers
+----------------------------------------
+
+When you create a new ASP.NET web app in Visual Studio, it adds "Microsoft.AspNet.Tooling.Razor" to the *project.json* file. This is the package that adds Tag Helper tooling. 
+
+Consider writing an HTML ``<label>`` element. As soon as you enter ``<l`` in the Visual Studio editor, IntelliSense displays matching elements:
+
+.. image:: intro/_static/label.png 
+
+Not only do you get HTML help, but the icon (the "@" symbol with "<>" under it).
+
+.. image:: intro/_static/tagSym.png 
+
+identifies the element as targeted by Tag Helpers. Pure HTML elements (such as the ``fieldset``) display the "<>"icon. 
+
+A pure HTML ``<label>`` tag displays the HTML tag (with the default Visual Studio color theme) in a brown font, the attributes in red, and the attribute values in blue.
+
+.. image:: intro/_static/LableHtmlTag.png 
+
+After you enter ``<label``, IntelliSense lists the available HTML/CSS attributes and the Tag Helper-targeted attributes:
+
+.. image:: intro/_static/labelattr.png
+
+IntelliSense statement completion allows you to enter the tab key to complete the statement with the selected value:
+
+.. image:: intro/_static/stmtcomplete.png
+
+As soon as a Tag Helper attribute is entered, the tag and attribute fonts change. Using the default Visual Studio "Blue" or "Light" color theme, the font is bold purple. If you're using the "Dark" theme the font is bold teal. The images in this document were taken using the default theme.
+
+.. image:: intro/_static/labelaspfor.png
+
+You can enter the Visual Studio *CompleteWord* shortcut (Ctrl +spacebar is the `default <https://msdn.microsoft.com/en-us/library/da5kh0wa.aspx>`__) inside the double quotes (""), and you are now in C#, just like you would be in a C# class. IntelliSense displays all the methods and properties on the page model. In the image below, I'm editing the ``Register`` view, so the ``RegisterViewModel`` is available.
```

Comment:
Clarify `IntelliSense displays all the methods and properties on the page model` to state that it is because the property is of type `ModelExpression`